themes/default/content/blog/cloud-engineering-fuels-startup-innovation/index.md

```diff
@@ -0,0 +1,94 @@
+# if you don't already have one.
+authors:
+    - george-huang
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - cloud-engineering
+    - case-studies
+
+# See the blogging docs at https://github.com/pulumi/docs/blob/master/BLOGGING.md.
+# for additional details, and please remove these comments before submitting for review.
+---
+The story of how the cloud fuels startup innovation seems never ending. In the beginning, AWS birthed cloud computing with its first service, SQS, in 2004 and quickly released several additional services (like S3, EC2, and SimpleDB). From this innovation, startups flourished because they were able to build, experiment, and grow faster than before at much lower cost. Airbnb, Netflix, Zynga, and many more were born, and the rest is history. 
+ 
+Today, a new generation of startups is flourishing because of the cloud, but this time with modern cloud architectures that are distributed, API-driven, and more resilient and scalable than ever. Today’s startups have to get to market even faster and rapidly innovate in order to delight customers and carve out market share. Most startups understand the benefits of adopting the modern cloud to help them achieve this. However, their ability to reap these benefits for competitive advantage depends on how well they can harness the modern cloud. 
+ 
+Within the [cloud engineering]({{< relref "/cloud-engineering" >}}) community, we see several common patterns for harnessing the modern cloud. Some startups have teams of full stack developers who need to rapidly and safely deploy cloud infrastructure and applications at high velocity. Others might have a few infrastructure or platform engineers who need to enable other developers to easily use cloud infrastructure on a self-serve basis. Many of these teams started off using domain-specific languages (DSLs) to manage infrastructure as code and quickly found that these languages were the limiting factor in achieving faster velocity. DSLs are cumbersome to use and don’t support the logic and expressiveness needed to build and manage modern architectures that are more complex in nature. DSL’s are also a barrier to entry to most developers. 
+ 
+In contrast, cloud engineering makes cloud infrastructure more accessible and easier to use for everyone with tried and true software engineering practices and tools. It embraces the reality that [all software is now cloud software]({{< relref "/blog/future-of-cloud-engineering/" >}}). A core practice is using a commonly used language like TypeScript, Python, Go, or C#. That’s one reason why more and more startups are choosing Pulumi to help them achieve competitive advantage through cloud infrastructure. It also doesn’t hurt that Pulumi was [rated the “Top Paying Technology” by developers]({{https://insights.stackoverflow.com/survey/2021#top-paying-technologies-other-tools}}) in 2021. 
```

Comment:
Along with the space, I removed a trailing slash at the end of the internal link. Hugo, our static website generator that builds the website, doesn't like trailing slashes in the `relref` (relative reference, as in relative to our site's domain) call, and it won't generate the link properly.

```suggestion
In contrast, cloud engineering makes cloud infrastructure more accessible and easier to use for everyone with tried and true software engineering practices and tools. It embraces the reality that [all software is now cloud software]({{< relref "/blog/future-of-cloud-engineering" >}}). A core practice is using a commonly used language like TypeScript, Python, Go, or C#. That’s one reason why more and more startups are choosing Pulumi to help them achieve competitive advantage through cloud infrastructure. It also doesn’t hurt that Pulumi was [rated the “Top Paying Technology” by developers]({{https://insights.stackoverflow.com/survey/2021#top-paying-technologies-other-tools}}) in 2021.
```